static/src/js/util/request/nlpSearchRequest.js

```diff
@@ -0,0 +1,207 @@
+import axios from 'axios'
+import { pick } from 'lodash-es'
+import { booleanClockwise, simplify } from '@turf/turf'
+
+import CmrRequest from './cmrRequest'
+import { getEarthdataConfig } from '../../../../../sharedUtils/config'
+import { convertNlpTemporalData } from '../temporal/convertNlpTemporalData'
+import { MAX_POLYGON_SIZE } from '../../constants/spatialConstants'
+
+/**
+ * Simplifies NLP geometry if it has too many points using Turf.js
+ * @param {Object | null} geometry - GeoJSON geometry object to simplify
+ * @returns {Object | null} Simplified geometry or null if simplification failed
+ */
+const simplifyNlpGeometry = (geometry) => {
```

Comment:
I'm not sure why this is a different function from when we simplify shapefiles, which are also geojson.

Reply:
The `simplifyShape` function expects an OpenLayers SimpleGeometry object as input, not the pure GeoJSON that
  comes back from NLP spatial responses. And making `simplifyShape` accept either inputs would add a lot of complexity to it.